libkpod/stop.go

```diff
@@ -11,18 +11,22 @@ func (c *ContainerServer) ContainerStop(container string, timeout int64) (string
 	if err != nil {
 		return "", errors.Wrapf(err, "failed to find container %s", container)
 	}
+	ctrID := ctr.ID()
 
 	cStatus := c.runtime.ContainerStatus(ctr)
+	if cStatus.Status == oci.ContainerStatePaused {
+		return "", errors.Errorf("cannot stop paused container %s", ctrID)
+	}
 	if cStatus.Status != oci.ContainerStateStopped {
```

Comment:
This can also be an else-if

Reply:
Unless you've strong opinions on this one, I'm going to leave as is.  If the first if fires, this one should never fire due to the return in the first if.

Reply:
It shaves off one line of code because we can put the else-if on the same line as the closing brace 😄 
But no, I don't feel too strongly about this

Reply:
I like the idea of switch statements.

Reply:
alrighty, switched to a switch